ui/store/identity.js

```diff
@@ -0,0 +1,18 @@
+import { writable } from "svelte/store";
+
+// TODO: define & structure this store
+
+// e.g. does each value gets its own store...
+//    const handle = "cloudhead" // as writable();
+//    const shareableEntityIdentifier = cloudhead@ViJQHAdeZoiEbaE5vv83dpjEun.rad" // as writable();
```

Comment:
In my understanding it makes sense to have a separate store for each value in Svelte.

If you store an object in a Svelte store, then each time there is an attribute update, the whole object will have to be re-written, which will trigger all associated components to update. This might not perform/look well.

Reply:
Then again, it might become unwieldy if we have a gazillion stores and don't have consistent name-spacing. Maybe it doesn't matter if a given object is anyway updated via one back-end query in one swoop.

In other words: I haven't figured out what the best practice is in this regard for Svelte 😬

Reply:
Interesting - will update to have one value per store. Will also make a research issue for collecting knowledge & questions about best practices 😊